### PR TITLE
Update Template 7 - PSU Consent Adoption

### PR DESCRIPTION
**Pull request by DinithiDiaz**
## Purpose
This PR is to update Template 7 - PSU Consent Adoption on Generating Reports page according to https://docs.google.com/document/d/1SNaTqFa5yvZEJuRolkj0Ugv3_B8dJW4_Wxmj3Hw1pUU/edit#.

This will fix [#532](https://github.com/wso2/docs-open-banking/issues/532).

### CODE REVIEW

File: en/docs/learn/data-publishing-reports.md [hunk 1]
```diff
@@ -113,10 +113,12 @@ following reports according to the OBIE templates:
 | 6 | Long-lived consents - Outstanding consents BoP | LONG_LIVED_CONSENT_ADOPTION | |
 | 7 | Long-lived consents - Revoked consents | LONG_LIVED_CONSENT_ADOPTION | |
 | 8 | Long-lived consents - Expired consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 9 | Long-lived consents - Renewed consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 10 | Long-lived consents - New consents | LONG_LIVED_CONSENT_ADOPTION | |
-| 11 | Long-lived consents - Outstanding consents EoP | LONG_LIVED_CONSENT_ADOPTION | |
-| 12 | Version | LONG_LIVED_CONSENT_ADOPTION | |
+| 9 | Long-lived consents - Consents Requiring Reauthentication | CONSENT_REQ_REAUTHENTICATION | This is the total number of unique Long-lived consents which required 90 days PSU re-authentication. Join ACCOUNTS_RAW_DATA and API_INVOCATION_RAW_DATA tables. And get the CONSENT_ID count AS TOTAL_CONSENT_REQ_REAUTH where (CONSENT_TYPE = 'long-lived' and AUTHORISATION_STATUS = 'Authorised' and RE_AUTHORISATION == false) having (EXPIRATION_DATE_TIME > REPORTING_DATE_TIME and REPORTING_DATE_TIME - 90 days> AUTHORIZED_TIME_DATE) |
```
Comment (isharailanga) [inline en/docs/learn/data-publishing-reports.md:116]: This seems like an SQL query:

Join ACCOUNTS_RAW_DATA and API_INVOCATION_RAW_DATA tables. And get the CONSENT_ID count AS TOTAL_CONSENT_REQ_REAUTH where (CONSENT_TYPE = 'long-lived' and AUTHORISATION_STATUS = 'Authorised' and RE_AUTHORISATION == false) having (EXPIRATION_DATE_TIME > REPORTING_DATE_TIME and REPORTING_DATE_TIME - 90 days> AUTHORIZED_TIME_DATE) 

should we rephrase this in a better way?wdyt

File: en/docs/learn/data-publishing-reports.md [hunk 2]
```diff
@@ -113,10 +113,12 @@ following reports according to the OBIE templates:
...
+| 10 | Long-lived consents - Renewed consents | LONG_LIVED_CONSENT_ADOPTION | |
+| 11 | Long-lived consents - New consents | LONG_LIVED_CONSENT_ADOPTION | |
+| 12 | Long-lived consents - Outstanding consents EoP | LONG_LIVED_CONSENT_ADOPTION | |
+| 13 | Long-lived consents - Total Refreshed consents | REFRSHED_CONSENT | This is the total number of unique Long-lived consents which have been refreshed (Re-authenticated). Join ACCOUNTS_RAW_DATA and API_INVOCATION_RAW_DATA tables. Get the CONSENT_ID count AS TOTAL_CONSENT_REQ_REAUTH where (CONSENT_TYPE = 'long-lived' and AUTHORISATION_STATUS = 'Authorised' and RE_AUTHORISATION == true) having (EXPIRATION_DATE_TIME > REPORTING_DATE_TIME and REAUTHORIZED_DATE_TIME > AUTHORIZED_TIME_DATE + 90 days) |
```
Comment (isharailanga) [inline en/docs/learn/data-publishing-reports.md:120]: I think we can use 'that' here instead. wdyt? 
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which

Comment (isharailanga) [inline en/docs/learn/data-publishing-reports.md:120]: 'Join ACCOUNTS_RAW_DATA and API_INVOCATION_RAW_DATA tables. Get the CONSENT_ID count AS TOTAL_CONSENT_REQ_REAUTH where (CONSENT_TYPE = 'long-lived' and AUTHORISATION_STATUS = 'Authorised' and RE_AUTHORISATION == true) having (EXPIRATION_DATE_TIME > REPORTING_DATE_TIME and REAUTHORIZED_DATE_TIME > AUTHORIZED_TIME_DATE + 90 days) |'

Same as above. Feels like an SQL query.

File: en/docs/learn/data-publishing-reports.md [hunk 3]
```diff
@@ -176,4 +198,4 @@ following reports according to the OBIE templates:
 - Additional Metrics - Revocation Notifications from ASPSPs
 - Additional Metrics - AIS Corporate Account Access (OPTIONAL)
 - Additional Metrics - Payment Refunds
-- Additional Metrics - 90 Days Re-authentication
\ No newline at end of file
+- Additional Metrics - 90 Days Re-authentication_
```
Comment (isharailanga) [inline en/docs/learn/data-publishing-reports.md:201]: Is the underscore here necessary?  `Additional Metrics - 90 Days Re-authentication_`

File: en/docs/learn/data-publishing-reports.md [hunk 4]
```diff
@@ -108,15 +108,37 @@ following reports according to the OBIE templates:
...
+| 7 | Long-lived consents - Revoked consents | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 8 | Long-lived consents - Expired consents | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 9 | Long-lived consents - Consents Requiring Reauthentication | `CONSENT_REQ_REAUTHENTICATION` | This is the total number of unique Long-lived consents, which required 90 days PSU re-authentication. <br/> <br/> See the note below this table to learn how to capture data. |
+| 10 | Long-lived consents - Renewed consents | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 11 | Long-lived consents - New consents | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 12 | Long-lived consents - Outstanding consents EoP | `LONG_LIVED_CONSENT_ADOPTION` | |
+| 13 | Long-lived consents - Total Refreshed consents | `REFRSHED_CONSENT` | This is the total number of unique Long-lived consents that have been refreshed (Re-authenticated). <br/> <br/> See the note below this table to learn how to capture data. |
+| 14 | Version | `LONG_LIVED_CONSENT_ADOPTION` | |
+
+!!! note "To capture Long-lived consents - Consents Requiring Re-authentication:"
```
Comment (isharailanga) [inline en/docs/learn/data-publishing-reports.md:123]: Lets format the field name

File: en/docs/learn/data-publishing-reports.md [hunk 5]
```diff
@@ -108,15 +108,37 @@ following reports according to the OBIE templates:
...
+       1. Join `ACCOUNTS_RAW_DATA` and `API_INVOCATION_RAW_DATA` tables.
+       2. Get the total count of `CONSENT_ID` as `TOTAL_CONSENT_REQ_REAUTH`. 
+       3. Consider the following conditions:
+          - `CONSENT_TYPE` = `'long-lived'`
+          - `AUTHORISATION_STATUS` = `'Authorised'`
+          - `RE_AUTHORISATION` == `false`
+          - `EXPIRATION_DATE_TIME` > `REPORTING_DATE_TIME`
+          - `REPORTING_DATE_TIME` - 90 days> `AUTHORIZED_TIME_DATE`
+
+!!! note "To capture Long-lived consents - Total Refreshed consents:"
```
Comment (isharailanga) [inline en/docs/learn/data-publishing-reports.md:133]: Lets format the field name

**Review by isharailanga — APPROVED** [review]

File: en/docs/learn/data-publishing-reports.md (hunk 2 above)
Comment (DinithiDiaz) [reply en/docs/learn/data-publishing-reports.md]: Fixed in f0a4ef558a5b877a1a92b590ea7c2e8ad534070c.

File: en/docs/learn/data-publishing-reports.md (hunk 1 above)
Comment (DinithiDiaz) [reply en/docs/learn/data-publishing-reports.md]: Fixed in 34b67885e2c5e89115e1466d8b58ad4f07742a8e.

File: en/docs/learn/data-publishing-reports.md (hunk 2 above)
Comment (DinithiDiaz) [reply en/docs/learn/data-publishing-reports.md]: Fixed in 34b67885e2c5e89115e1466d8b58ad4f07742a8e.

File: en/docs/learn/data-publishing-reports.md (hunk 4 above)
Comment (DinithiDiaz) [reply en/docs/learn/data-publishing-reports.md]: Fixed in fae1a3d4d37ff71f5893786f015c13654abdc2d9.

File: en/docs/learn/data-publishing-reports.md (hunk 5 above)
Comment (DinithiDiaz) [reply en/docs/learn/data-publishing-reports.md]: Fixed in fae1a3d4d37ff71f5893786f015c13654abdc2d9.

File: en/docs/learn/data-publishing-reports.md (hunk 3 above)
Comment (DinithiDiaz) [reply en/docs/learn/data-publishing-reports.md]: Fixed in fae1a3d4d37ff71f5893786f015c13654abdc2d9.

**Pull request merged**